docs/DeveloperGuide.adoc

```diff
@@ -723,15 +723,29 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
+|`* * *` |user |I want to have the ability to create favourite lists |view my most important contacts
```

Comment:
Note that the column header is "So that I can...". Hence, the content in the third column needs to be rephrased.

Reply:
Noted. Edited accordingly.